DRAFT.md

```diff
@@ -715,6 +715,24 @@ constexpr indirect<T, Alloc>& operator->() noexcept;
 
 Otherwise, the interface of the specialization is as defined in [optional].
 
+#### X.Y.13 Formatter support [indirect.fmt]
+
+```c++
+// [indirect.fmt]
+template <class T, class Alloc, class charT>
+  requires { std::formatter<T, char>{}; };
```

Comment:
@jbcoe does this look like how a requirement should be specified?  Note in the implementation I created a concept for this, but here I've used an inline requires of the same check.

Reply:
I don’t think that the standard uses concept requirements in code. Just text in a _Requires_ clause

Reply:
I've added this as a precondition that "The specialization formatter<T, charT> meets the Formatter requirements.".  This requirement is mentioned in https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1636r2.pdf